includes/templates/bootstrap/templates/tpl_modules_checkout_address_book.php

```diff
@@ -17,7 +17,12 @@
  */
 require DIR_WS_MODULES . zen_get_module_directory('checkout_address_book.php');
 while (!$addresses->EOF) {
-    if ($addresses->fields['address_book_id'] == $_SESSION['sendto']) {
+    $selected = ($addresses->fields['address_book_id'] == $_SESSION['sendto']); 
+    if ($current_page_base == 'checkout_payment_address') {
```

Comment:
As identified in the associated zencart repository, this needs to use `===` and `FILENAME_CHECKOUT_PAYMENT_ADDRESS`.